src/Chainweb/Version.hs

```diff
@@ -40,6 +40,7 @@ module Chainweb.Version
 , window
 -- ** Date-based Transaction Disabling
 , txSilenceEndDate
+, txActivationDate
```

Comment:
I am not sure if `Chainweb.Version` is the right place for this constant, since it's not a property of the version. Normally, things in this file stay forever.

It's probably fine to define it here for convenience. But it needs a comment (that can be identified with `grep`) that explain that it can be removed after Dec 6.

Reply:
I *will* be a property of the Version very soon. You'll notice that in the other PR, this is the chief value used to turn everything else on.